src/commands/cron/ListCrons.ts

```diff
@@ -25,7 +28,7 @@ export class ListCrons extends BotgartCommand {
         assertType(responsible, "User");
         assertType(guild, "Guild");
         if(!responsible) {
-            log("error", "Can not execute lscron without member to reply to. Canceling.");
+            LOG.error("Can not execute lscron without member to reply to. Canceling.")
```

Comment:
; 
okay, there seems to be some sort of pattern, so it was probably a concious decision to omit semicolons here. Personally, I am not a big fan of omitting either semicolons or braces just because you can do it. As always, there are heated arguments regarding what is right or wrong on this topic https://mislav.net/2010/05/semicolons/ but since most of the existing code base is semicolon-afflicated, I would argue in favour of using semicolons consistently.